<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="Raghav_Global" failures="1" tests="4" name="Smoke Test Execution" time="11.692" errors="0" timestamp="2025-09-12T10:57:16 IST">
  <testcase classname="api.test.UserTests" name="testCreateUser" time="3.579"/>
  <testcase classname="api.test.UserTests" name="testDeleteUser" time="2.696"/>
  <testcase classname="api.test.UserTests2" name="testCreateMultipleUsers" time="2.393"/>
  <testcase classname="api.test.UserTests2" name="testDeleteUser" time="2.641">
    <failure type="java.lang.AssertionError" message="Failed to delete user! expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Failed to delete user! expected [200] but found [404]
at api.test.UserTests2.testDeleteUser(UserTests2.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
  <testcase classname="api.test.UserTests2" name="testUpdateUser" time="0.0">
    <ignored/>
  </testcase> <!-- testUpdateUser -->
  <testcase classname="api.test.UserTests" name="testUpdateUser" time="0.0">
    <ignored/>
  </testcase> <!-- testUpdateUser -->
  <testcase classname="api.test.DDTests" name="testPostUser" time="0.0">
    <ignored/>
  </testcase> <!-- testPostUser -->
  <testcase classname="api.test.UserTests2" name="testGetAllUsers" time="0.0">
    <ignored/>
  </testcase> <!-- testGetAllUsers -->
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="0.0">
    <ignored/>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.DDTests" name="deleteAllUsers" time="0.0">
    <ignored/>
  </testcase> <!-- deleteAllUsers -->
</testsuite> <!-- Smoke Test Execution -->
